<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupchatwindow</class>
 <widget class="QWidget" name="groupchatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelGroupName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GroupName</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>201</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="statusNewer">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>499</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检查在线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>9</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>增加成员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>9</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入到文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backgroundSetButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>背景</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sndButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
